cognite/client/data_classes/simulators/runs.py

```diff
@@ -56,3 +61,183 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        )
+
+    def dump(self, camel_case: bool = True) -> dict[str, Any]:
+        output = super().dump(camel_case=camel_case)
+        if self.inputs is not None:
+            output["inputs"] = [input_.dump(camel_case=camel_case) for input_ in self.inputs]
+        return output
+
+    def as_write(self) -> SimulationRunWrite:
+        return self
+
+
+class SimulationRun(SimulationRunCore):
+    """
+    Every time a simulation routine executes, a simulation run object is created.
+    This object ensures that each execution of a routine is documented and traceable.
+    Each run has an associated simulation data resource, which stores the inputs and outputs of a
+    simulation run, capturing the values set into and read from the simulator model to ensure
+    the traceability and integrity of the simulation data.
+
+    Simulation runs provide a historical record of the simulations performed, allowing users to analyze
+    and compare different runs, track changes over time, and make informed decisions based on the simulation results.
+
+    This is the read/response format of a simulation run.
+
+    Args:
+        id (int): The id of the simulation run
+        simulator_external_id (str): External id of the associated simulator
+        simulator_integration_external_id (str): External id of the associated simulator integration
+        model_external_id (str): External id of the associated simulator model
+        model_revision_external_id (str): External id of the associated simulator model revision
+        routine_revision_external_id (str): External id of the associated simulator routine revision
+        routine_external_id (str): External id of the associated simulator routine
+        run_type (str): The type of the simulation run
+        status (str): The status of the simulation run
+        data_set_id (int): The id of the dataset associated with the simulation run
+        user_id (str): The id of the user who executed the simulation run
+        log_id (int): The id of the log associated with the simulation run
+        created_time (int): The number of milliseconds since epoch
+        last_updated_time (int): The number of milliseconds since epoch
+        status_message (str | None): The status message of the simulation run
+        simulation_time (int | None): Simulation time in milliseconds. Timestamp when the input data was sampled. Used for indexing input and output time series.
+        run_time (int | None): Run time in milliseconds. Reference timestamp used for data pre-processing and data sampling.
+
+    """
+
+    def __init__(
+        self,
+        id: int,
+        simulator_external_id: str,
+        simulator_integration_external_id: str,
+        model_external_id: str,
+        model_revision_external_id: str,
+        routine_revision_external_id: str,
+        routine_external_id: str,
+        run_type: str,
+        status: str,
```

Comment:
@erlendvollset these two properties could be an enum but we're already past the 500 line limit, we'll address this in another PR.

Reply:
What about just making it a literal? That is what we typically do for status

```suggestion
        status: Literal["ready", "running", "success", 'failure"],
```